packages/monaco/src/browser/monaco-editor-model.ts

```diff
@@ -79,6 +81,8 @@ export class MonacoEditorModel implements ITextEditorModel, TextEditorDocument {
 
     protected resourceVersion: ResourceVersion | undefined;
 
+    protected EditorOptions = monaco.editor.EditorOption;
```

Comment:
Please move this to the top, below imports.

Reply:
`import EditorOptions = monaco.editor.EditorOption;`

Reply:
Got it. Thanks!